src/app/main/component/user/components/profile/profile-dashboard/profile-dashboard.component.ts

```diff
@@ -86,20 +95,45 @@ export class ProfileDashboardComponent implements OnInit, OnDestroy {
     });
   }
 
-  initGetUserEvents(): void {
+  getUserLocation() {
+    navigator.geolocation.getCurrentPosition((position) => {
+      if (position) {
+        this.userLatitude = position.coords.latitude;
+        this.userLongitude = position.coords.longitude;
+      }
+    });
+  }
+
+  onCheckboxChange(checkboxNumber: number) {
+    if (checkboxNumber === 1) {
```

Comment:
Using numbers to identify elements's not usually a good practice because it decreases readability. At least, with this logic, we can pass and use 'ONLINE' or 'OFFLINE'